Framework/PythonInterface/test/python/plugins/algorithms/LoadWANDSCDTest.py

```diff
@@ -63,6 +67,40 @@ def test(self):
 
         LoadWANDTest_ws.delete()
 
+    def test_withNorm(self):
+        # create van data
+        van = LoadWANDSCD('HB2C_7000.nxs.h5')
+        van.setSignalArray(np.full_like(van.getSignalArray(), 25))
+        van.setErrorSquaredArray(np.full_like(van.getSignalArray(), 25))
+        AddSampleLog(van, LogName='time', LogText='42', LogType='Number Series', NumberType='Double')
```

Comment:
These log values you are adding for the tests are wrong, it should be `duration` and `monitor_count`.